Resources/Prototypes/Access/misc.yml

```diff
@@ -17,6 +17,7 @@
   - Engineering
   - Medical
   - Mercenary # Frontier
+  - Pilot # Frontier
```

Comment:
Do we even need a full new rank just for pilot?
Also you are missing mention of it on the C# level i think
Look at what was done on the Merc update.

Reply:
done :(